Sources/CoreFoundation/internalInclude/CoreFoundation_Prefix.h

```diff
@@ -109,6 +109,11 @@ typedef char * Class;
 #include <pthread.h>
 #endif
 
+#if TARGET_OS_ANDROID
+#define HAVE_STRLCPY 1
+#define HAVE_STRLCAT 1
+#endif
```

Comment:
I think that we should consider doing this via the build configuration.

Reply:
I think we've been doing this in the headers because build configuration is spread across several systems (cmake, SPM), and they do not all support the same features.

Reply:
Instead of setting this separately here, can you just [add `TARGET_OS_ANDROID` to the list below](https://github.com/swiftlang/swift-corelibs-foundation/blob/21b3196b33a64d53a0989881fc9a486227b4a316/Sources/CoreFoundation/internalInclude/CoreFoundation_Prefix.h#L203)? That's how I do it on my CI.